tgmusicbot.py

```diff
@@ -210,7 +211,7 @@ def make_squarethumb(thumbnail, output):
     original_thumb = Image.open(thumbnail)
     squarethumb = _crop_to_square(original_thumb)
     squarethumb.thumbnail((TG_THUMB_MAX_LENGTH, TG_THUMB_MAX_LENGTH),
-                          Image.ANTIALIAS)
+                          Image.Resampling.LANCZOS)
```

Comment:
What's the differences?

Reply:
Pillow says it's the same, just ANTIALIAS will be deprecated in future